### ISSUE TITLE
Usage of the ontology with W3C Verifiable Credentials

### ISSUE THREAD

**Issue by tplooker** [fhircat/FHIRCat#40]
Hi, firstly thanks for this project it is great work! I work in the [Credentials Community Group](https://w3c-ccg.github.io/community/) working with technologies like [W3C Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) and as of recent there has been much interest in applications of the technology for vaccination certificates. As a result I have begun the initiative to define a vocabulary for [this purpose](https://w3id.org/vaccination) however ideally we would like to leverage more existing ontologies like HL7 FHIR so are open to redefining terms in the vocab to suit this and I believe we could leverage some of the great work you guys have done to date to accomplish this?

A couple of quick questions w.r.t your work

1. How actively is the ontology maintained and how complete is it? It would be great to know this in reference to both R4 and R5
2. It appears the current IRIs the context uses are not resolvable i.e I get a 404 on http://hl7.org/fhir/Patient.gender

**Comment by ericprud** [comment]
1. The vocab is generated from StructureDefinitions, so not maintained by us but by the thousands of folks tooling on FHIR resources.
2. We're working more closely with the build process now. If you don't mind the names changing, we can probably ground those URLs in GETable resources. If we want to keep those URLs, the question becomes more political 'cause they populate primary real-estate. OTOH, what else would go there but information about Patient gender?

**Comment by tplooker** [comment]
Thanks for the prompt responses.

> We're working more closely with the build process now. If you don't mind the names changing, we can probably ground those URLs in GETable resources. If we want to keep those URLs, the question becomes more political 'cause they populate primary real-estate. OTOH, what else would go there but information about Patient gender?

IMO to get the most out of semantic technologies the term IRIs should dereference to some human readable information about what they are so in regards to your example of patient gender I would have expected to land [here](http://hl7.org/fhir/patient-definitions.html#Patient.gender)

**Comment by ericprud** [comment]
yeah, that makes sense. it means we don't have any versioning, but we haven't been great about that anyways.
If we used `http://hl7.org/fhir/patient-definitions#Patient.gender` (no `.html` extension), we could conceivably negotiate to an RDF representation some day in the futuer.
